.github/workflows/main.yml

```diff
@@ -15,19 +15,23 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        build: [linux-release, windows-release]
+        build: [linux-release, windows-release, osx-release]
         include:
           - build: linux-release
             os: ubuntu-latest
             config: release
+          - build: osx-release  
+            os: macos-14 # https://github.blog/changelog/2024-01-30-github-actions-introducing-the-new-m1-macos-runner-available-to-open-source/
+            config: release            
           - build: windows-release
             os: windows-2019
             config: release
     steps:
     - uses: actions/checkout@v4
     - uses: actions/setup-dotnet@v4
       with:
-        dotnet-version: | 
+        dotnet-version: |
+          6.0.x
           7.0.x
```

Comment:
Should we drop 7.0? The official end-of-support is in about a month.

Reply:
My opinion would be to use only NET8. So I will make this.

Reply:
BTW, should we drop 7.0 in our main package if the official support ends in the future?